admin/metrics/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='OSFStatistic',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('users', models.IntegerField(default=0, verbose_name=b'OSF users')),
+                ('delta_users', models.IntegerField(default=0)),
+                ('unregistered_users', models.IntegerField(default=0, verbose_name=b'Unregistered users')),
+                ('projects', models.IntegerField(default=0, verbose_name=b'Number of projects')),
+                ('delta_projects', models.IntegerField(default=0)),
+                ('public_projects', models.IntegerField(default=0, verbose_name=b'Number of public projects')),
+                ('delta_public_projects', models.IntegerField(default=0)),
+                ('registered_projects', models.IntegerField(default=0, verbose_name=b'Number of projects registered')),
+                ('delta_registered_projects', models.IntegerField(default=0)),
+                ('date', models.DateTimeField(default=None)),
```

Comment:
> I am beginning to realize just how tough it is to keep up-to-date remotely.

Yeah, it takes some intentionality. GitLab has been putting out [good material](https://about.gitlab.com/2016/03/23/remote-communication/) on this lately. :-)

> This comes from conversations with Sara and Matt Spitzer on how we need to drastically overhaul how we track metrics

I'm not going to get hung up on the specifics here. Sounds like you are in conversation with the right people about the specific stats we need.

> I will discuss this today to see if they would rather this be moved to a proof-of-concept' ticket while the rest of metrics is nailed down.

With the framework implemented on this PR, we can revise the specific stats we track in future PRs.